prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -53,110 +53,110 @@
  *   rxr_pkt_init_ctrl() uses init functions declared in rxr_pkt_type.h
  */
 static
-int rxr_pkt_init_ctrl(struct rxr_ep *rxr_ep, int entry_type, void *x_entry,
+int rxr_pkt_init_ctrl(struct rxr_ep *rxr_ep, int entry_type, struct rxr_op_entry *x_entry,
 		      int ctrl_type, struct rxr_pkt_entry *pkt_entry)
 {
```

Comment:
This change to the interface may require callers to cast the pointer to the correct type in some cases.  Did you look for these or see any compiler warnings?

Reply:
Yes. There's only one caller that already uses the correct type. Line 294 same file.